### ISSUE TITLE
RigL TF2 on Resnet50 + Imagenet

### ISSUE THREAD

**Issue by AmoghDabholkar** [google-research/rigl#12]
I wanted to use RigL TF2 code to train a sparsified Resnet50 architecture and see how that goes.
I loaded a Resnet50 -
    model = tf.keras.applications.resnet50(
    include_top=True,
    weights="imagenet",
    input_tensor=None,
    input_shape=None,
    pooling=None,
    classes=1000,
    classifier_activation="softmax",)
    
And then sparsified it.
The test accuracy seems to be stuck at 0.1% for the first 2000 steps even
Is this a common problem?

**Comment by evcu** [comment]
I recommend using the tf-1 code for resnet-50 training as the tf2-code provided in this repo is not multi-worker. Note that rigl starts from random weights and I don't know what would happen if you randomly prune a pretrained checkpoint.
